hummingbot/templates/conf_dev_4_twap_strategy_TEMPLATE.yml

```diff
@@ -8,14 +8,14 @@ strategy: null
 # twap strategy
 
 # Exchange and token parameters
-exchange: null
-market: null
+connector: null
```

Comment:
Is there a reason for changing this to `connector`? 
Also considering that there are changes to the configuration, It would be necessary to change the `template_version` number, so that the bot will prompt users for the new configurations.

Reply:
I decided to change the configuration key from exchange to connector to make it easier for the user to identify it. I had to change the prompt text displayed to request that information, for the completer to only offer the spot connectors. The prompt now asks for the connector instead of the exchange.

I have changed the version number. Thank you for pointing that out.